controllers/cleanup.go

```diff
@@ -213,13 +210,14 @@ func (r *VolumeSnapshotRestoreReconciler) CleanRestoreResources(log logr.Logger)
 		return false, err
 	}
 
-	vsr.Status.Phase = volsnapmoverv1alpha1.SnapMoverRestorePhaseCompleted
-	now := metav1.Now()
-	vsr.Status.CompletionTimestamp = &now
+	// Update VSR status as completed
+	if vsr.DeletionTimestamp.IsZero() {
 
```

Comment:
Why have we removed marking the `vsr.Status.CompletionTimestamp` ?

Reply:
Logic moved to replicationDestination controller: https://github.com/migtools/volume-snapshot-mover/blob/master/controllers/replicationdestination.go#L189-L190